src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
> When Im reading your comments, I feel like you are taking good care about the implementation side of the interface, not the user side

Because what we're designing is not a user-facing library, but an interoperability layer.

Of course we should (and we will) have all the comforts and utility and ease of use you seek - I want that as well... just not at this level, but in user-facing libraries that consume this interoperability layer, providing everyone with the freedom to judge, experiment and discover what is their individual idea of comfort and utility.

> could you write a (non-inline) comment what your current suggestion for stream factories are?

Where should I write it?

See [this reply](https://github.com/http-interop/http-factory/pull/16/files/281c55057916cee53cf3ac1812dec68bc0b78fe3#r76677311) by @weierophinney and my reply immediately following that - this was where it crystallized for me.